pool/web/view/page/page_admin_custom_field_groups.ml

```diff
@@ -0,0 +1,173 @@
+                       [ div
+                           [ txt
+                               (field
+                               |> name
+                               |> Name.find_opt language
+                               |> CCOption.map_or ~default:"-" Name.value_name)
+                           ]
+                       ; div
+                           [ input
+                               ~a:
+                                 [ a_input_type `Hidden
+                                 ; a_name
+                                     Message.Field.(CustomField |> array_key)
+                                 ; a_value (field |> id |> Id.value)
+                                 ]
+                               ()
+                           ]
+                       ; div
+                           ~a:
+                             [ a_class [ "flexrow"; "flex-gap"; "align-center" ]
+                             ]
+                           [ a
+                               ~a:
+                                 [ a_href
+                                     (Url.Field.edit_path (model field, id field)
+                                     |> Sihl.Web.externalize_path)
+                                 ]
+                               [ txt
+                                   Pool_common.(
+                                     Message.More
+                                     |> Utils.control_to_string language)
+                               ]
+                           ]
+                       ])
+                   fields)
+            ; submit_element
+                language
+                Message.UpdateOrder
+                ~submit_type:`Success
+                ()
+            ]
+        ]
+  in
+  div
+    ~a:[ a_class [ "stack-lg" ] ]
+    [ form
+        ~a:
+          [ a_method `Post
+          ; a_action (Sihl.Web.externalize_path action)
+          ; a_class [ "stack-lg" ]
+          ]
+        [ csrf_element csrf ()
+        ; div
+            ~a:[ a_class [ "stack" ] ]
+            [ h4
+                ~a:[ a_class [ "heading-4" ] ]
+                [ txt
+                    Pool_common.(
+                      Message.Field.Name
+                      |> Utils.field_to_string language
+                      |> CCString.capitalize_ascii)
+                ]
+            ; div ~a:[ a_class [ "stack" ] ] name_inputs
+            ]
+        ; submit_element
+            language
+            Message.(
+              let field = Some Field.CustomField in
+              match custom_field_group with
+              | None -> Create field
+              | Some _ -> Update field)
+            ~submit_type:`Success
+            ()
+        ]
+    ; sort_fields_form
+    ]
+;;
+
+let detail
+  ?custom_field_group
+  current_model
+  (Pool_context.{ language; _ } as context)
+  sys_langauges
+  flash_fetcher
+  =
+  let title =
+    Pool_common.(
+      Utils.control_to_string
+        language
+        Message.(
```

Comment:
similar code to handler

Reply:
Not sure if we should pass titles (or similar) from the handler to the views if we can determine it there. But as this is repeating, I could create a utils function in the Component module to create the titles.